lib/reline/ansi.rb

```diff
@@ -278,7 +284,7 @@ def self.cursor_pos
         buf = @@output.pread(@@output.pos, 0)
         row = buf.count("\n")
         column = buf.rindex("\n") ? (buf.size - buf.rindex("\n")) - 1 : 0
-      rescue Errno::ESPIPE
+      rescue Errno::ESPIPE, IOError
```

Comment:
For my learning: Why did you need `IOError`?

Reply:
It is because `pread` raises IOError in some situation #537.
Removed this change because it is out of scope of this pull request.